cmd/installer/cli/join_printcommand.go

```diff
@@ -3,18 +3,39 @@ package cli
 import (
 	"context"
 	"fmt"
+	"os"
 
 	"github.com/replicatedhq/embedded-cluster/cmd/installer/kotscli"
 	"github.com/replicatedhq/embedded-cluster/pkg/runtimeconfig"
+	rcutil "github.com/replicatedhq/embedded-cluster/pkg/runtimeconfig/util"
 	"github.com/spf13/cobra"
 )
 
 func JoinPrintCommandCmd(ctx context.Context, name string) *cobra.Command {
+	var rc runtimeconfig.RuntimeConfig
+
 	cmd := &cobra.Command{
 		Use:   "print-command",
 		Short: fmt.Sprintf("Print controller join command for %s", name),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if os.Getuid() != 0 {
+				return fmt.Errorf("update command must be run as root")
+			}
```

Comment:
Do we need this? If so the msg should be changed `s/update/join`.

Reply:
changed